src/test/java/org/zalando/nakadi/service/EventPublisherTest.java

```diff
@@ -128,6 +139,147 @@ public void whenValidationFailsThenSubsequentItemsAreAborted() throws Exception
     }
 
     @Test
+    public void whenEventIsTooLargeThenResultIsAborted() throws Exception {
```

Comment:
There are two tests with multibyte characters already, lines 252-280. The first one is an event that is just 1 byte over the limit; the second one is exactly at the limit. Multibyte events are created on lines 502-512